unifi/DOCS.md

```
@@ -162,37 +160,18 @@ check [the contributor's page][contributors].
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
+See [LICENSE][license]
 
 [addon-badge]: https://my.home-assistant.io/badges/supervisor_addon.svg
-[addon]: https://my.home-assistant.io/redirect/supervisor_addon/?addon=a0d7b954_unifi&repository_url=https%3A%2F%2Fgithub.com%2Fhassio-addons%2Frepository
-[contributors]: https://github.com/hassio-addons/addon-unifi/graphs/contributors
+[addon]: https://my.home-assistant.io/redirect/supervisor_addon/?addon=fe9ae0f9_unifi&repository_url=https%3A%2F%2Fgithub.com%2Fblaubaer%2Fha-addon-unifi
+[contributors]: https://github.com/blaubaer/ha-addon-unifi/graphs/contributors
 [discord-ha]: https://discord.gg/c5DvZ4e
 [discord]: https://discord.me/hassioaddons
```

Markdownlint

Unused link or image reference definition: "discord"
Link and image reference definitions should be needed

(MD053, link-image-reference-definitions)